integration_tests/test_gravity_2.py

```diff
@@ -0,0 +1,304 @@
+import pytest
```

Comment:
move test to another file because it seems any additional test added to test_gravity fails